src/main/java/com/shipmentEvents/demo/EventHandler.java

```diff
@@ -0,0 +1,174 @@
+package com.shipmentEvents.handlers;
+retur
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;retur&#34;, delete this token